src/executables/util_cs.c

```diff
@@ -4540,3 +4541,98 @@ flashback (UTIL_FUNCTION_ARG * arg)
 #endif /* !CS_MODE */
 
 }
+
+int
+memmon (UTIL_FUNCTION_ARG * arg)
+{
+#if defined(CS_MODE)
+  UTIL_ARG_MAP *arg_map = arg->arg_map;
+  char er_msg_file[PATH_MAX];
+  const char *database_name;
+  bool need_shutdown = false;
+  const char *outfile_name;
+  FILE *outfile_fp = NULL;
+  int error_code = NO_ERROR;
+  MMON_SERVER_INFO server_info;
+
+  outfile_name = utility_get_option_string_value (arg_map, MEMMON_OUTPUT_S, 0);
+
+  database_name = utility_get_option_string_value (arg_map, OPTION_STRING_TABLE, 0);
+  if (database_name == NULL)
+    {
+      goto print_memmon_usage;
+    }
+
+  if (check_database_name (database_name))
+    {
+      goto error_exit;
+    }
+
+  if (outfile_name)
+    {
+      outfile_fp = fopen (outfile_name, "w+");
+    }
+  else
+    {
+      goto print_memmon_usage;
```

Comment:
In this case, it seems more natural to output to stdout.

Reply:
I agree. Done.